measurementkit/mkcgo/mkcgo.go

```diff
@@ -23,19 +23,17 @@ import (
 	// #cgo windows,amd64 CFLAGS: -I/usr/local/opt/mingw-w64-measurement-kit/include/
 	// #cgo windows,amd64 LDFLAGS: /usr/local/opt/mingw-w64-measurement-kit/lib/libmeasurement_kit.a
 	// #cgo windows,amd64 LDFLAGS: /usr/local/opt/mingw-w64-libmaxminddb/lib/libmaxminddb.a
-	// #cgo windows,amd64 LDFLAGS: /usr/local/opt/mingw-w64-curl/lib/libcurl.a
 	// #cgo windows,amd64 LDFLAGS: /usr/local/opt/mingw-w64-libevent/lib/libevent_openssl.a
 	// #cgo windows,amd64 LDFLAGS: /usr/local/opt/mingw-w64-libressl/lib/libssl.a
 	// #cgo windows,amd64 LDFLAGS: /usr/local/opt/mingw-w64-libressl/lib/libcrypto.a
 	// #cgo windows,amd64 LDFLAGS: /usr/local/opt/mingw-w64-libevent/lib/libevent_core.a
 	// #cgo windows,amd64 LDFLAGS: /usr/local/opt/mingw-w64-libevent/lib/libevent_extra.a
 	// #cgo windows,amd64 LDFLAGS: -lws2_32
+	// #cgo windows,amd64 LDFLAGS: -fstack-protector-strong
```

Comment:
This is inherited by LibreSSL, which forces this flag. It's super annoying because this flag is not allowed by CGO by default and, to be able to use it, we need to `export CGO_LDFLAGS_ALLOW='-fstack-.*'`